draft-ietf-ppm-dap.md

```diff
@@ -1719,11 +1697,11 @@ where:
 * `public_share` is the report's public share
 * `plaintext_input_share` is the Leader's `PlaintextInputShare`
 
-The methods are defined in {{Section 5.8 of !VDAF}}. This process determines
-the initial per-report `state`, as well as the initial `outbound` message to
-send to the Helper. If `state` is of type `Rejected`, then the report is
+`ping_pong_leader_init` is defined in {{Section 5.7.1 of !VDAF}}. This process
+determines the initial per-report `state`, as well as the initial `outbound`
+message to the Helper. If `state` is of type `Rejected`, then the report is
```

Comment:
VDAF section reference was wrong. I have a note somewhere to audit all the VDAF section references to make sure they're correct for the current draft.